test/testcafe/spec/ChallengeOktaVerifyTotp_spec.js

```diff
@@ -52,6 +52,12 @@ test
     await t.expect(await challengeOktaVerifyTOTPPageObject.getTotpLabel())
       .contains('Enter code from Okta Verify app');
 
+    // Verify links
+    await t.expect(await challengeOktaVerifyTOTPPageObject.switchAuthenticatorLinkExists()).ok();
+    await t.expect(challengeOktaVerifyTOTPPageObject.getSwitchAuthenticatorLinkText()).eql('Verify with something else');
+    await t.expect(await challengeOktaVerifyTOTPPageObject.signoutLinkExists()).ok();
+    await t.expect(challengeOktaVerifyTOTPPageObject.getSignoutLinkText()).eql('Back to sign in');
```

Comment:
Looking at this again, I think we have an opportunity to condense these tests. I created a follow-up for us to look into breaking out the footer checking/content logic out to a shared assertion ([OKTA-402002](https://oktainc.atlassian.net/browse/OKTA-402002)).